clash-lib/src/Clash/Driver.hs

```diff
@@ -383,7 +388,14 @@ generateHDL reprs domainConfs bindingsMap hdlState primMap tcm tupTcm typeTrans
         then writeEdam hdlDir (topNm, varUniq topEntity) deps edamFiles0 fileNames
         else pure (edamFiles0, fileNames)
 
-      writeManifest manPath manifest0{fileNames=fileNames1}
+      -- If we are generating (System)Verilog, we could potentially verilate
+      -- the results. Clash can output a C++ shim for doing this automatically.
+      fileNames2 <-
+        case hdl of
+          VHDL -> pure fileNames1
+          _ -> writeVerilatorShim hdlDir topNm fileNames1
+
+      writeManifest manPath manifest0{fileNames=fileNames2}
```

Comment:
I'd love for this to become a separate executable someday. It feel icky to have this all dumped in `Driver`.